doc/source/user_guide/merging.rst

```diff
@@ -206,10 +206,9 @@ behavior:
 
 .. warning::
```

Comment:
I think the warning can be removed, as it is not that important any more (now we are past 1.0)

Reply:
Ok, thx. Is gone